api/websocket/server/relay.go

```diff
@@ -124,11 +125,44 @@ func (ws *WsServer) sendInboundRelayMessage(relayMessage *pb.Relay) {
 				sigChainLen: int(relayMessage.SigChainLen),
 			})
 		}
-	} else {
+		if time.Duration(relayMessage.MaxHoldingSeconds) > pongTimeout/time.Second {
+			ok := ws.messageDeliveredCache.Push(relayMessage)
+			if !ok {
+				log.Warningf("MessageDeliveredCache full, discarding messages.")
+			}
+		}
```

Comment:
Why still cache msg even sendInboundMessage success?

Reply:
`sendInboundMessage` does not mean the client will definitely receive message when there is a proxy between node and client.